portal/hackathons/views.py

```diff
@@ -267,6 +268,12 @@ def post(self, request, *args, **kwargs):
                     user=user, hackathon=self.object
                 )
 
+        # Update graduation eligibility status of user
+        elif new_status == "closed" and cur_status == "complete":
```

Comment:
`closed` is the starting status and `complete` is the final status (where everyone can submit as an individual).
The status never goes from `closed` to `complete`.

The final transition is from `submissions_closed` to `complete`.

Reply:
cool, thanks, I thought we would put it back to close.

Reply:
Updated with the right status @hcastilho :)